chromium_src/check_chromium_src_config.json5

```diff
@@ -62,11 +62,6 @@
         "chrome/browser/bookmarks/android/bookmark_bridge.cc": [
             "BraveBookmarkBridge",
         ],
-        "chrome/browser/browsing_data/chrome_browsing_data_remover_delegate.cc": [
-            "CHROME_BROWSER_WEB_APPLICATIONS_COMMANDS_CLEAR_BROWSING_DATA_COMMAND_H_",
-            "CHROME_BROWSER_WEB_APPLICATIONS_WEB_APP_PROVIDER_H_",
-            "CHROME_BROWSER_WEB_APPLICATIONS_WEB_APP_UTILS_H_",
-        ],
```

Comment:
Was this particular change intentional? I don't see how it's relevant to this PR, but I could definitely be missing something. Asking because `check_chromium_src.py` is now failing with the following messages:

```[Error] chromium_src:
  Override chromium_src\chrome\browser\browsing_data\chrome_browsing_data_remover_delegate.cc
  defines symbol CHROME_BROWSER_WEB_APPLICATIONS_COMMANDS_CLEAR_BROWSING_DATA_COMMAND_H_ but the symbol could not be found in
  C:\work\brave-browser\src\chrome\browser\browsing_data\chrome_browsing_data_remover_delegate.cc
  and is not used internally in the override.
-------------------------
[Error] chromium_src:
  Override chromium_src\chrome\browser\browsing_data\chrome_browsing_data_remover_delegate.cc
  defines symbol CHROME_BROWSER_WEB_APPLICATIONS_WEB_APP_PROVIDER_H_ but the symbol could not be found in
  C:\work\brave-browser\src\chrome\browser\browsing_data\chrome_browsing_data_remover_delegate.cc
  and is not used internally in the override.
-------------------------
[Error] chromium_src:
  Override chromium_src\chrome\browser\browsing_data\chrome_browsing_data_remover_delegate.cc
  defines symbol CHROME_BROWSER_WEB_APPLICATIONS_WEB_APP_UTILS_H_ but the symbol could not be found in
  C:\work\brave-browser\src\chrome\browser\browsing_data\chrome_browsing_data_remover_delegate.cc
  and is not used internally in the override.
```

Reply:
yes it was mistake, returned it back with PR: https://github.com/brave/brave-core/pull/24624